.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: Repository (in owner/repo format)
+        required: true
+        default: kiali/kiali.io
+        type: string
```

Comment:
IMO, you should remove this.

In Jenkins, it was needed the repository parametrized, because it is an external system and it needs to know what repo to work with. However, in GHactions it must always be the repo firing the action.